modules/heat_transfer/src/fvbcs/FVRadiativeHeatFluxBCBase.C

```diff
@@ -18,7 +18,11 @@ FVRadiativeHeatFluxBCBase::validParams()
   params.addCoupledVar("temperature", "temperature variable");
   params.addParam<Real>("stefan_boltzmann_constant", 5.670367e-8, "The Stefan-Boltzmann constant.");
   params.addParam<FunctionName>(
-      "Tinfinity", "0", "Temperature of the body in radiative heat transfer.");
+      "Tinfinity_function", "Function for the temperature of the body in radiative heat transfer.");
+  params.addParam<MooseFunctorName>(
+      "Tinfinity_functor", "Functor for the temperature of the body in radiative heat transfer.");
+  params.addParam<PostprocessorName>(
+      "Tinfinity_pp", "Postprocessor for the temperature of the body in radiative heat transfer.");
```

Comment:
postprocessors can actually be used as functors, so I think you can just go with the functor parameter. You could also probably delete the function parameter too since functions can be used as functors

Reply:
Then the name of the functor parameter (the only remaining one) can be `Tinfinity`